dvc/repo/get.py

```diff
@@ -17,6 +19,30 @@
 logger = logging.getLogger(__name__)
 
 
+def _is_git_file(repo, path):
+    if not os.path.isabs(path):
+        try:
+            output = repo.find_out_by_relpath(path)
+            if not output.use_cache:
+                return True
+        except OutputNotFoundError:
+            return True
+    return False
+
+
+def _copy_git_file(repo, src, dst):
+    src_full_path = os.path.join(repo.root_dir, src)
+    dst_full_path = os.path.abspath(dst)
+
+    try:
+        if os.path.isdir(src_full_path):
+            shutil.copytree(src_full_path, dst_full_path)
+        else:
+            shutil.copy2(src_full_path, dst_full_path)
+    except FileNotFoundError:
```

Comment:
Weird that this doesn't fail (probably because not tested, eh? 🙂 ), but FileNotFoundError doesn't exist in py2, so you need to get it from dvc.utils.compat.

Reply:
https://github.com/iterative/dvc/pull/2837/files#diff-3d39da458b4c00949c13695a2b5615b6R153

Reply:
@danihodovic Indeed, I'm wrong, sorry.